src/App.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
+import LandingPageCard from './components/Card/LandingPageCard';
 
 function App() {
   return (
     <div>
       <h1 className="text-3xl text-purple-500">Train To Maintain</h1>
+      <LandingPageCard />
```

Comment:
you should pass props here
```suggestion
      <LandingPageCard image={"image URL here"} title={"title gores here"}/>
```